go/test/endtoend/onlineddl/scheduler/onlineddl_scheduler_test.go

```diff
@@ -1183,6 +1191,36 @@ func testScheduler(t *testing.T) {
 		})
 	})
 	// in-order-completion
+	t.Run("in-order-completion: multiple drops for nonexistent tables and views, sequential", func(t *testing.T) {
+		u, err := schema.CreateOnlineDDLUUID()
+		require.NoError(t, err)
+
+		sqls := []string{
+			fmt.Sprintf("drop table if exists t4_%s", u),
+			fmt.Sprintf("drop view  if exists t1_%s", u),
+			fmt.Sprintf("drop table if exists t2_%s", u),
+			fmt.Sprintf("drop view  if exists t3_%s", u),
+		}
+		sql := strings.Join(sqls, ";")
+		var vuuids []string
+		t.Run("drop multiple tables and views, in-order-completion", func(t *testing.T) {
+			uuidList := testOnlineDDLStatement(t, createParams(sql, ddlStrategy+" --in-order-completion", "vtctl", "", "", true)) // skip wait
+			vuuids = strings.Split(uuidList, "\n")
+			assert.Equal(t, 4, len(vuuids))
```

Comment:
If you didn't already know, you can use `assert.Len` too/instead.

Reply:
I did not know!

Reply:
done